hardware/libhost/devices.h

```diff
@@ -3,7 +3,7 @@
 
 #include "xcl2/xcl2.hpp"
 
-namespace alveo {
+// namespace alveo {
```

Comment:
Please keep this namespace. There are other families of devices such as Versal.

Reply:
u280 is in this space. I find that if I don't comment out this namespace, u280 will not work.

Reply:
> Please keep this namespace. There are other families of devices such as Versal.
u280 is in this space. I find that if I don't comment out this namespace, u280 will not work.